core/modules/catalog/helpers/index.ts

```diff
@@ -4,8 +4,10 @@ import { calculateProductTax } from '../helpers/tax'
 import flattenDeep from 'lodash-es/flattenDeep'
 import omit from 'lodash-es/omit'
 import remove from 'lodash-es/remove'
-import groupBy from 'lodash-es/groupBy'
+import reduce from 'lodash-es/reduce'
```

Comment:
Imho we can just ES features instead of using lodash to reduce bundle size :)

Reply:
Makes sense, I'll take care of it!